content/about.md

```diff
@@ -3,7 +3,32 @@ title: "About"
 date: "2021"
 ---
 
-### External links
+Or is an experienced SRE for over a decade, building companies and their infrastructure.
+Responsible for educating engineers working with cloud-native platforms on best-practices,
+helping them be the best and most efficient they can be in order to provide value with least friction.
+Designing highly scalable platforms that serves billions of requests and happy customers with resiliency and simplicity in mind.
+
+Manifesting systems and bringing them to life with the best in class technology-stack that fits the needs of the business.
+Whether it be Kubernetes, Google Cloud Platform, AWS, Go, Prometheus etc.
+Writing packages alongside developers to help them abstract the complexity away from servers they build
```

Comment:
Taking part in open-sourcing tools and utilities we developed internally for public usage.